Hunted Mobile/Hunted Mobile/Hunted Mobile/Repository/GameRepository.cs

```diff
@@ -20,6 +21,12 @@ public class GameRepository {
             return response.IsSuccessful;
         }
 
+        public string GetLogoUrl(int gameId) {
+            var url = HttpClientRequestService.GetUrl($"games/{gameId}/logo");
+
+            return url;
```

Comment:
Het zou mooi zijn als dit een Resource terug geeft. Maar dan zou de functie denk ik in de ResourceRepository moeten komen.